codalab/model/tables.py

```diff
@@ -74,6 +74,20 @@
   sqlite_autoincrement=True,
 )
 
+# Stores information about the files, directories and links stored in the
+# bundle.
+bundle_contents_index = Table(
+  'bundle_contents_index',
+  db_metadata,
+  Column('bundle_uuid', String(63), ForeignKey(bundle.c.uuid), nullable=False),
+  Column('path', Text, nullable=False),
+  Column('type', String(63), nullable=False),
+  Column('size', Integer, nullable=False),
+  Column('perm', Integer, nullable=False),
+  Column('link', Text, nullable=True),
```

Comment:
These are well documented in index_contents.

Also, I'm going to push back on this request. It's going to take too much of my time, and I don't see it as being worth it.